Source/WebKit/UIProcess/RemoteLayerTree/mac/RemoteLayerTreeEventDispatcher.cpp

```diff
@@ -651,6 +657,21 @@ void RemoteLayerTreeEventDispatcher::handleSyntheticWheelEvent(PageIdentifier pa
 {
     ASSERT_UNUSED(pageID, m_pageIdentifier == pageID);
 
+    auto scrollingTree = this->scrollingTree();
+    if (scrollingTree->scrollingPerformanceTestingEnabled()) {
+        if (event.momentumPhase() == WebWheelEvent::PhaseBegan)
+            WTFBeginSignpostAlways(nullptr, ScrollingPerformanceTestMomentumInterval, "isAnimation=Yes;");
+
+        if (!m_ScrollingPerformanceTestSignpostEndIntervals && !std::abs(event.delta().height())) {
+            m_ScrollingPerformanceTestSignpostEndIntervals++;
+            WTFEndSignpostAlways(nullptr, ScrollingPerformanceTestScrollInterval, "isAnimation=YES;");
+            WTFEndSignpostAlways(nullptr, ScrollingPerformanceTestMomentumInterval, "isAnimation=Yes;");
```

Comment:
Why is the capitalization of YES different on these two lines?

Reply:
Typo. Fixing now